src/pages/home/Search.tsx

```diff
@@ -100,7 +101,7 @@ function Search() {
       setSearchTerm('')
       setError('location', {
         type: 'manual',
-        message: 'Search a Location',
+        message: Constants.search_a_location,
```

Comment:
After the above changes this would be 
CommonConstants.SEARCH_LOCATION

You can change all other texts as like as.